jqModal.js

```diff
@@ -370,7 +378,8 @@
 			toTop: false,
 			onShow: onShow,
 			onHide: onHide,
-			onLoad: false
+			onLoad: false,
+			blockMsgClass: 'blockMsg'
```

Comment:
great, I've used `blockMsgClass` to let it compatible to [jquery blockUI](http://www.malsup.com/jquery/block/), but your idea look much better